pkg/adapter/metrics.go

```diff
@@ -27,11 +30,12 @@ const (
 
 // Label kinds supported by mixer.
 const (
-	String LabelKind = iota
+	String LabelType = iota
```

Comment:
Agree that I think we will ultimately want a separation between the representations, especially as we add a broader set of adapters.